### ISSUE TITLE
Collider documentation is hard to follow

### ISSUE THREAD

**Issue by gd2shoe** [joe7575/techage#148]
I watched your video [here](https://youtu.be/8Q_l_vCf61w).  Without this, I'm not sure I ever would have gotten the collider working.  

The issue is the isobutane coolant system and the lack of a usable diagram.  Each magnet has 2 input ports, but there's no indication that only one of the two ports needs to be used for each magnet; that each magnet is to be treated a bit like a tank.  With a lack of information to go by, and a bunch of magnets with two ports on them, it looked to me like I needed to daisy-chain the magnets together, alternating inside and outside the ring.  Within that context, "pump the isobutane into the tube system" didn't help me figure out where to attach the pump.  I had a tube system.  It wasn't the correct tube system, but it was a tube system, and it wasn't accepting isobutane.  This line also baffled me: "In addition, cooling is required, which must also be installed at the detector. Isobutane is required for cooling."  This makes it sound like the isobutane somehow gets injected at the heat exchanger.

I knew I had a fundamental misunderstanding, and I reread the documentation several times over as carefully as I could.  I just couldn't find anything that would jog me out of my rut.

I think this documentation could use some review.  In particular, it would have helped me tremendously to have had a line somewhere that made it explicitly known that the magnets only needed piping on one side or the other to accept isobutane.

**Comment by joe7575** [comment]
- I removed the isobutane input port on the back and improved (hopefully) the manual.
